Capgemini.Xrm.CdsDataMigrator/Capgemini.Xrm.CdsDataMigratorLibrary/UserControls/ImportPage.cs

```diff
@@ -83,6 +83,13 @@ string IImportPageView.CrmMigrationToolSchemaPath
             set => fisSchemaFile.Value = value;
         }
 
+        decimal IImportPageView.maxThreads
```

Comment:
Were there any limits to this value before? Min value: 1, Max value: ?

Reply:
> Thanks @danroche10
> 
> A nice easy change 😄. Just a minor comment and a question.

@tdashworth - made requested changes!